avae/train.py

```diff
@@ -82,22 +86,46 @@ def train(
         params=vae.parameters(), lr=learning  # , weight_decay=1e-5
     )
 
-    beta_arr = KLD_weight(
+    print(epochs, cyc_method_beta, beta_min, beta_max, beta_cycle, beta_ratio)
```

Comment:
Is this a leftover debug print statement or useful feedback? If latter, can we add some text to explain what's being printed?

Reply:
thanks for pointing this out, I always miss one :))